### PR TITLE
[EMCAL-526] Initial version of the RawTask

### PR DESCRIPTION
**Pull request by cterrevo**
Monitor objects
- payload size
- error counter
- maximum ADC per SM
- mean ADC per SM
- ADC RMS per SM

current version requires RDH version 4

### CODE REVIEW

File: Modules/EMCAL/src/RawTask.cxx [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+      uint64_t currentTrigger(0);
+      bool first = true; //for the first event
+      short int maxADCSM[20];
+      while (rawreader.hasNext()) {
+        rawreader.next();
+        auto payLoad = rawreader.getPayload();
+        auto payLoadSizeTest = payLoad.getPayloadWords().size() * sizeof(uint32_t); //payloadsize in byte
+
+        auto headerR = rawreader.getRawHeader();
+        mPayloadSizePerDDL->Fill(headerR.feeId, payLoadSizeTest / 1024.);
```
Comment (mfasDa) [inline Modules/EMCAL/src/RawTask.cxx]: The PR https://github.com/AliceO2Group/AliceO2/pull/2710 has just been merged. One can now simply get the payload size via  `rawreader.getPayloadSize()`.

Comment (cterrevo) [reply Modules/EMCAL/src/RawTask.cxx]: Thanks Markus for your comments! I have updated the code with your suggestions!

**Comment by mfasDa** [comment]
@Barthelemy The PR should be ready now. Could you please merge (squash merge) once the build tests pass? Thanks!

**Comment by Barthelemy** [comment]
@mfasDa Ok !

@MichaelLettrich It seems that we have a problem with the build machines ?

**Comment by MichaelLettrich** [comment]
I will investigate

**Comment by mfasDa** [comment]
@Barthelemy The build tests are now passing - could you please merge? Thanks in advance!

**Comment by Barthelemy** [comment]
Yes, just had a look at the code, all good.

**Pull request merged**